GG/src/ListBox.cpp

```diff
@@ -431,12 +426,18 @@ void ListBox::Row::SetColAlignments(const std::vector<Alignment>& aligns)
     if (aligns == m_col_alignments)
         return;
 
-    m_col_alignments = aligns;
+    m_col_alignments.resize(m_cells.size(), ALIGN_NONE);
+
+    for (std::size_t i = 0; i < m_cells.size(); ++i) {
```

Comment:
although the resize() above makes m_col_alignments.size() the same as m_cells,size(), it sure seems to me like this would read better by using m_col_alignments.size() here.

Reply:
ty, fixed